client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useState, useCallback, useRef } from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Suggestions from '../../../../components/suggestions';
```

Comment:
For better or worse, `client` is a module directory in Calypso, so typically you'll see client-relative imports like this:

```suggestion
import Suggestions from 'components/suggestions';
```

That said, before landing this we should go through the work of extracting the suggestion component to `@automattic/components`.